app/controllers/DayAggregateEventController.scala

```diff
@@ -47,8 +46,9 @@ trait DayAggregateEventController extends BaseController {
   }
 
   private def find(location: Option[Location], skills: Seq[SkillType] = Nil) = {
-    eventsRepository.getEvents(None, None, location, skills).map {
-      _.groupBy(e => DayAggregateEvent(e.date, e.location)).keys.toList
+    eventsRepository.getEvents(None, None, location, skills).map { events =>
+      events.distinctTransform(_.date, e => DayAggregateEvent(e.date, Location("Home"))) ++
```

Comment:
Not verbose. I don't think distinctTransform function needed. It took me a while to understand what is that.  Basically you can do groupBy date and then map to DAE with Home Location.

Reply:
Do we need to put Home value in yaml file? There are places in code where we validate Venues/locations against  available values there.

Reply:
Home is a valid location now, and there are places across the code base where the input location is validated against the yaml file so will throw exceptions if it's not there.